frontend/src/v5/ui/routes/viewer/theme.ts

```diff
@@ -100,13 +100,25 @@ export const theme = createTheme(
 						},
 					},
 				},
+				MuiFormControl: {
+					styleOverrides: {
+						root: {
+							margin: 0,
+						},
+					},
+				},
 				MuiTextField: {
 					styleOverrides: {
 						root: {
 							margin: 0,
 							'& .MuiInputBase-root input': {
 								lineHeight: 22,
-								height: 24,
+								height: 26,
```

Comment:
This line causes text to be cut off in the Links list when adding attachments
![image](https://user-images.githubusercontent.com/62481250/192498120-5ac207a0-1695-4018-9738-d84fdd7cc3e6.png)